service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/database/AzureDatabaseUtilsRunner.java

```diff
@@ -163,8 +163,8 @@ public void pgDumpDatabase(
 
     runAzureDatabaseUtils(
         azureCloudContext,
-        sourceWorkspaceId,
-        createPodDefinition(sourceWorkspaceId, podName, envVars, secretStringData),
+        destinationWorkspaceId,
+        createPodDefinition(destinationWorkspaceId, podName, envVars, secretStringData),
```

Comment:
the changes in `pgDumpDatabase` are the only meaningful ones. the rest are renamed variables.